editor/editor_properties.cpp

```diff
@@ -1211,6 +1211,7 @@ void EditorPropertyLayers::_button_pressed() {
 void EditorPropertyLayers::_menu_pressed(int p_menu) {
 	if (p_menu == grid->layer_count) {
 		ProjectSettingsEditor::get_singleton()->popup_project_settings();
+		ProjectSettingsEditor::get_singleton()->clear_filter();
```

Comment:
Maybe this can be a boolean flag to `popup_project_settings()` instead?

Reply:
done